src/react/projects/spark-core-react/src/SprkDictionary/SprkDictionary.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+const SprkDictionary = (props) => {
+    const { keyValuePairs, variant, idString, additionalClasses, ...other } = props;
+    const classNames = classnames(
+        'sprk-c-Dictionary',
+        additionalClasses, {
+            'sprk-c-Dictionary--striped': variant === 'striped'
+        },
+    );
+
+    const keyValuePairsMap = Object.keys(keyValuePairs).map(function (term) {
+        return (
+            <div className="sprk-c-Dictionary__keyvaluepair">
+                <dt className="sprk-c-Dictionary__key sprk-b-Label sprk-b-Label--no-input">{term}</dt>
+                <dd className="sprk-c-Dictionary__value">{keyValuePairs[term]}</dd>
+            </div>
+        )
+    });
+
+    return (
+        <div className={classNames} data-id={idString}>
+            <dl className="sprk-c-Dictionary__keyvaluepairs">
+                {keyValuePairsMap}
+            </dl>
+        </div>
+    );
+}
+
+SprkDictionary.propTypes = {
+    // The string to use for the data-id attribute
+    keyValuePairs: PropTypes.object.isRequired,
+    // The link variant that determines the class names
+    variant: PropTypes.oneOf(['striped']),
```

Comment:
If there is only one variant, should this property be replaced with something like "isStriped"?

Reply:
at the moment, angular is just "striped"

How do you value prepending "is" as a benefit?

Reply:
have a boolean/flag called isStriped. So the usage would be

`<SprkDictionary idString="dictionary1" keyValuePairs={keyValuePairs} isStriped />`

instead of 

`<SprkDictionary variant="striped" idString="dictionary1" keyValuePairs={keyValuePairs} />`

Reply:
so I'm wondering about this and future friendliness. What if there's a new variant? Then we would cause a breaking change.